app/core/components/FallbackSpinner.tsx

```diff
@@ -0,0 +1,34 @@
+import { Center, Flex, Spinner, Text } from "@chakra-ui/react"
```

Comment:
the spinner is really annoying as is, so added a skeleton header and centered it